### ISSUE TITLE
Difference in replayed warc with pywb and browsertrix

### ISSUE THREAD

**Issue by adityaraj-28** [webrecorder/browsertrix-crawler#333]
I am aware that browsertrix uses pywb in the background. I tried a website `https://www.kugou.com/` and noticed some notable missing elements with browsertrix.
I started pywb using 
`docker run -e INIT_COLLECTION=myarc -p 8080:8080 -v ./pywb-data/:/webarchive webrecorder/pywb wayback --record --live -a`
And hit google chrome browser with `http://localhost:8080/myarc/record/https://www.kugou.com/`, I waited for the page to load, scrolled and waited for the images to load after scrolling
Here's a screenshot of replayed warc with [replayweb](https://replayweb.page/)
<img width="1209" alt="Screenshot 2023-06-20 at 7 39 37 PM" src="https://github.com/webrecorder/browsertrix-crawler/assets/118821295/c471c708-e6e5-4842-9cf7-97852bf8bede">

With browsertrix I started it using
`docker run -v $PWD/crawls:/crawls/ webrecorder/browsertrix-crawler crawl --url "https://www.kugou.com/"  --generateWARC --combineWARC true --collection kuguo1  --scopeType page-spa --waitUntil networkidle0 --timeout 100  --behaviorTimeout 100 --netIdleWait 30`

When replaying the combined warc with pywb I got the following warc replay
<img width="1209" alt="Screenshot 2023-06-20 at 7 58 01 PM" src="https://github.com/webrecorder/browsertrix-crawler/assets/118821295/4e242141-ec12-4956-94d4-a9d6890d6631">

Is there any configurational error I am doing when running using browsertrix. Can anyone please help me figure out whats the issue.

**Comment by lakshya16240** [comment]
I am able to reproduce this issue. Even tried using `--waitUntil load,networkidle0` as was suggested in one of the previous issues. Another similar website on which the issue can be replicated is `https://www.cctv.com/index.shtml`. PFA the screenshots.


**Actual Website**
<img width="1683" alt="Screenshot 2023-06-20 at 8 14 54 PM" src="https://github.com/webrecorder/browsertrix-crawler/assets/20909956/6cd7a024-c4f2-49f0-a625-cece45383bc5">




**Using Pywb recorder **
<img width="1654" alt="Screenshot 2023-06-20 at 8 16 12 PM" src="https://github.com/webrecorder/browsertrix-crawler/assets/20909956/defbe91d-3be0-4dac-b72e-a2d488db1b1b">



**Using browsertrix and replaying using replay.web**

<img width="1728" alt="Screenshot 2023-06-20 at 8 17 03 PM" src="https://github.com/webrecorder/browsertrix-crawler/assets/20909956/d7feea2b-80ba-4d5d-b660-1334a7a843d1">

**Comment by edsu** [comment]
I see the same behavior as well. Did you happen to notice a message like this in the log output? 

```
{"logLevel":"info","timestamp":"2023-06-20T15:03:50.679Z","context":"behaviorScript","message":"Behavior log","details":{"state":{"segments":1},"msg":"Skipping autoscroll, page seems to not be responsive to scrolling events","page":"https://www.kugou.com/","workerid":0}}
```

I did, and I think this is similar to #321 perhaps? It appears autoscrolling is not working on the page?

**Comment by adityaraj-28** [comment]
Yes, I did get this. And as autoscrolling is not happening, and the images load lazily after scroll, it isn't able to record those assets. Interesting

**Comment by lakshya16240** [comment]
Absolutely. I see this log as well. Seems weird though, since some of the images appearing towards the end of scroll of the page have been loaded
<img width="1728" alt="Screenshot 2023-06-20 at 9 04 00 PM" src="https://github.com/webrecorder/browsertrix-crawler/assets/20909956/6fae67a5-6f7d-4a2e-883d-1d83917ddf16">

**Comment by lakshya16240** [comment]
Absolutely. I see this log as well. Seems weird though, since some of the images appearing towards the end of scroll of the page have been loaded
<img width="1728" alt="Screenshot 2023-06-20 at 9 04 00 PM" src="https://github.com/webrecorder/browsertrix-crawler/assets/20909956/6fae67a5-6f7d-4a2e-883d-1d83917ddf16">

**Comment by lakshya16240** [comment]
As rightly pointed out by @adityaraj-28, most of the lazy loaded images are not being crawled properly. But, there are indeed a few instances in which some of the loaded images are laze loaded, as can be seen in the image below.
<img width="1728" alt="Screenshot 2023-06-20 at 9 09 08 PM" src="https://github.com/webrecorder/browsertrix-crawler/assets/20909956/6e65da57-4854-4a9c-974c-a82ba09a4215">
